src/less/infotip/stories/infotip.stories.js

```diff
@@ -105,3 +105,29 @@ export const paragraphExpanded = () => `
         </span>
     </span>
 </p>`;
+
+export const infotipModal = () => `
+    <span class="infotip infotip--modal">
+        <button class="icon-btn icon-btn--transparent dialog-button" type="button" data-makeup-for="lightbox-dialog-mini-infotip-0" aria-expanded="false" aria-label="Help">
```

Comment:
How does the modal display here? For me, it looked like after the page was loaded, something caused it to open or maybe there was a delay of some sort in Storybook? 

We don't run custom `js` in Storybook, so it's probably ok, but the `button` has `aria-expanded="false", which doesn't seem consistent.

Since it's a new story, running Percy isn't necessary, but maybe we should run it for `infotip` just to make sure the modal is going to get captured properly.

Reply:
Dialogs by default are shown by default. 
If we add a `hidden` attribute then it is hidden. In this case there is no hidden attribute. I can update the button to be aria-expanded true to be consistent but it will show up regardless.